Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM node:5
 
 RUN apt-get install libpq-dev
 
-COPY package.json /
+WORKDIR /sequelize
+COPY package.json /sequelize/
 RUN npm install
 
-WORKDIR /sequelize
-VOLUME /sequelize
+COPY . /sequelize/
```

Comment:
Probably sqlite.

If we can fix the issue of having to have the same node version inside and outside of the container that would be nice :). As long as we can still mount /lib and /test as volumes so we dont need a rebuild when changing files